spec/models/response_csv_spec.rb

```diff
@@ -77,12 +81,12 @@
 
     let(:repeat_form) do
       create(:form,
-              question_types:
-                ["integer",
-                  {repeating: {q_types: ["text", "integer", "select_multiple"], name: "Fruit"}},
-                  "integer",
-                  {repeating: {q_types: ["text", "geo_multilevel_select_one",  "integer"], name: "Vegetable"}}
-                ]).tap do |f|
+      question_types:
+      ["integer",
+        {repeating: {q_types: ["text", "integer", "select_multiple"], name: "Fruit"}},
+        "integer",
+        {repeating: {q_types: ["text", "geo_multilevel_select_one",  "integer"], name: "Vegetable"}}
+        ]).tap do |f|
```

Comment:
I like getting rid of the huge indents, but our typical style would be to have these one level further in than line 83.